app/controllers/routes_controller.rb

```diff
@@ -1,13 +1,55 @@
 class RoutesController < ApplicationController
-  helper_method :routes
+  helper_method :routes, :route
 
   def index
     @model_attributes = model_attributes(Route)
```

Comment:
In `model_attributes` you already assign the result to the `@model_attributes` variable.
So this assignment is no needed here